apps/idam/idam-user-dashboard/preview.yaml

```diff
@@ -10,7 +10,7 @@ spec:
       replicas: 1
       ingressHost: idam-user-dashboard.service.core-compute-preview.internal
       environment:
-        STRATEGIC_SERVICE_URL: 'http://idam-api.preview.platform.hmcts.net'
-        STRATEGIC_PUBLIC_URL: 'http://idam-web-public.preview.platform.hmcts.net'
-        DASHBOARD_URL: 'http://idam-user-dashboard.preview.platform.hmcts.net'
-        OAUTH_CLIENT_REDIRECT: 'http://idam-user-dashboard.preview.platform.hmcts.net/oauth2/callback'
+        STRATEGIC_SERVICE_URL: 'https://idam-api.service.core-compute-preview.internal'
```

Comment:
I think (but might be wrong) that the recommendation from PlatOps is to use svc url in preview:
```
STRATEGIC_SERVICE_URL: 'http://idam-api'
STRATEGIC_PUBLIC_URL: 'http://idam-web-public'
```